src/pages/CourseFeedback/CourseFeedbackPage.jsx

```diff
@@ -0,0 +1,70 @@
+import { Button, TextField } from "@mui/material";
+import PageLayout from "../../components/PageLayout";
+import HorizontalLabeledSlider from "./components/HorizontalLabeledSlider";
+import { useState } from "react";
+import "./CourseFeedbackPage.css";
+
+/**
+ * The `CourseFeedbackPage` function is a React component that renders a form for users to provide
+ * feedback on a course, including sliders for course content and length, a text field for additional
+ * comments, and a submit button.
+ * @returns The CourseFeedbackPage component.
+ */
+function CourseFeedbackPage() {
+  const [courseContentValue, setCourseContentValue] = useState(0);
+  const [courseLengthValue, setCourseLengthValue] = useState(0);
+  const [additionalFeedback, setAdditionalFeedback] = useState("");
+
+  const handleCourseContentChange = (event) => {
+    setCourseContentValue(event.target.value);
+  };
+
+  const handleCourseLengthChange = (event) => {
+    setCourseLengthValue(event.target.value);
+  };
+
+  const handleTextChange = (event) => {
+    setAdditionalFeedback(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
```

Comment:
Can you add here a logging of the selected values to demonstrate the functionality of sending them
@NicolasKmbr

Reply:
I did before but I thought logs like that don't belong in the main. Should I add console.logs again?

Reply:
It is fine at the moment because they are just a placeholder for the following backend functionalities

Reply:
Added